news/_posts/core-weekly/2021-08-02-coreweekly-30-2021.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: post
+title:  "PrestaShop Core Weekly - Week 31 of 2021"
+subtitle: "An inside look at the PrestaShop codebase"
+date:   2021-08-02
+authors: [ PrestaShop ]
+image: /assets/images/2017/04/core_weekly_banner.jpg
+twitter_image: /assets/images/theme/banner-core-weekly.jpg
+icon: icon-calendar
+tags:
+ - core-weekly
+---
+
+This edition of the Core Weekly report highlights changes in PrestaShop's core codebase from Monday 2nd to Sunday 8th of August 2021.
```

Comment:
Encore raté ! 😄 

This one, Core Weekly fetched data from current week, not past week. Hahaha.